src/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx

```diff
@@ -90,29 +93,48 @@ const LimitedSupportedCluster: React.FC<LimitedSupportedClusterProps> = ({
   </TextContent>
 );
 
-const FullySupportedCluster: React.FC = () => (
+export const FullySupportedCluster: React.FC = () => (
   <>
     <CheckCircleIcon color={okColor.value} />
     &nbsp;Your installed cluster will be fully supported
   </>
 );
 
-export type ReviewClusterFeatureSupportLevelsProps = {
-  clusterFeatureSupportLevels: FeatureIdToSupportLevel | undefined;
+export const getFeatureSupportLevelTitle = (fullySupported: boolean): string => {
+  const supportLevel: string = fullySupported ? 'Full' : 'Limited';
+  return `Cluster support level: ${supportLevel}`;
 };
 
-const ReviewClusterFeatureSupportLevels: React.FC<ReviewClusterFeatureSupportLevelsProps> = ({
-  clusterFeatureSupportLevels,
+export const ClusterFeatureSupportLevelsDetailItem: React.FC<{ cluster: Cluster }> = ({
+  cluster,
 }) => {
-  if (!clusterFeatureSupportLevels) {
-    return null;
+  const featureSupportLevelData = React.useContext(FeatureSupportLevelContext);
+
+  const clusterFeatureSupportLevels = React.useMemo(() => {
+    return getLimitedFeatureSupportLevels(cluster, featureSupportLevelData);
+  }, [cluster, featureSupportLevelData]);
+
+  const fullySupported: boolean = React.useMemo<boolean>(() => {
+    if (!clusterFeatureSupportLevels) {
+      return false;
+    }
+    return isFullySupported(clusterFeatureSupportLevels);
```

Comment:
Why not making `isFullySupported` args optional and handle the logic above, returning false, inside that function? This is the only place this function is being used right now

Reply:
right. with these changes there's no need to share this code so I put all the logic for checking if a cluster is supported in the component